navigation/create_control/launch/create_control.launch

```diff
@@ -0,0 +1,18 @@
+<launch>
+
+  <!-- Load controller configuration from YAML file to parameter server -->
+  <arg name="real_robot" value="$(optenv REAL_ROBOT False)"/>
+  <arg name="robot_id"  default="$(optenv ID 1)"      doc="Unique identifier of the robot [1-Inf.)"/>
+  <arg name="ns"        value="create$(arg robot_id)" doc="Namespace of the robot. By default: create1"/>
+
+  <node name="robot_hw_interface" pkg="create_control" type="create_control_node" if="$(arg real_robot)"
+    ns="$(arg ns)">
+    <rosparam command="load" file="$(find ca_driver)/config/default.yaml"/>
+  </node>
+  <rosparam file="$(find create_control)/config/create_2_control.yaml" command="load"/>
+
+  <!-- load the controller -->
+  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
+    output="screen" ns="$(arg ns)" args="mobile_base_controller"/>
```

Comment:
Instead of adding `ns` in this launch file, you can edit `create_empty_world.launch` directly.

```suggestion
    output="screen" args="mobile_base_controller"/>
```